build.sh

```diff
@@ -315,9 +315,11 @@ if buildAll || hasArg libcudf; then
            LIBCUDF_FS=$(ls -lh ${LIB_BUILD_DIR}/libcudf.so | awk '{print $5}')
            MSG="${MSG}<br/>libcudf.so size: $LIBCUDF_FS"
         fi
-        echo "$MSG"
-        python ${REPODIR}/cpp/scripts/sort_ninja_log.py ${LIB_BUILD_DIR}/.ninja_log --fmt html --msg "$MSG" > ${LIB_BUILD_DIR}/ninja_log.html
-        cp ${LIB_BUILD_DIR}/.ninja_log ${LIB_BUILD_DIR}/ninja.log
+        BMR_DIR=${RAPIDS_ARTIFACTS_DIR:-"${LIB_BUILD_DIR}"}
+        echo "Metrics output dir: [$BMR_DIR]"
+        mkdir -p ${BMR_DIR}
+        python ${REPODIR}/cpp/scripts/sort_ninja_log.py ${LIB_BUILD_DIR}/.ninja_log --fmt html --msg "$MSG" > ${BMR_DIR}/ninja_log.html
+        cp ${LIB_BUILD_DIR}/.ninja_log ${BMR_DIR}/ninja.log
```

Comment:
```suggestion
        cp ${LIB_BUILD_DIR}/.ninja_log ${BMR_DIR}/ninja.txt
```

I don't know how this build metrics report is generally consumed.

If it's useful to be able to view it directly in the browser, you can change this file extension to `.txt`, which will allow it to be viewed in the browser.

Right now the file is downloaded when the link is clicked since `.log` is not a file extension that browsers recognize (or at least my Chrome browser doesn't recognize it).

Reply:
It is probably not very useful to see in the browser. Usually you run tools on it to get meaningful results so I think just download is ok for now.